.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Test
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+          node-version: [10.x, 12.x, 14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install Dependencies
+        run: npm install
+      - name: Bootstrap Packages
+        run: npm run setup-full
+      - name: Lint & Unit Tests
+        run: npm run test
+      - name: E2E Protocol Tests
+        if: ${{ matrix.node-version == '12.x' }}
+        run: npm run test:e2e
+      - name: E2E Firefox Launch Tests
+        if: ${{ matrix.node-version == '12.x' }}
+        run: |
+          sudo add-apt-repository ppa:ubuntu-mozilla-daily/ppa
+          sudo apt-get update
+          sudo apt-get install firefox-trunk
+          run: npm run test:e2e:firefox
+  e2e-windows:
```

Comment:
I suggest using a operating system matrix like here: https://github.com/erwinheitzman/find-duplicate-strings/blob/master/.github/workflows/github-ci.yml

Reply:
I don't want to run 3 jobs for 3 node versions on 3 operating systems though. It is sufficient to test all 3 node versions only in Linux

Reply:
@christian-bromann is it possible to have one Windows job? Please please!

Reply:
There is a windows job that checks if devtools works in Chrome

Reply:
Is it possible to run unit tests in windows as well?

Reply:
We used to have many times in the past some issues on Windows, like slashes in paths and others.
Assuming that most of us work in mac or linux it would be good to run tests in windows

Reply:
I can add unit tests for Windows 👍

Reply:
unit tests are currently broken, created an issue to fix them: https://github.com/webdriverio/webdriverio/issues/5950 ... I will disable them in the meantime.

Reply:
I'll take a look if I have some time, can't promise though